go/eth/rpc.go

```diff
@@ -13,29 +20,68 @@ type Signature struct {
 }
 
 // TODO(prin-r): Declare unexported variable that represents a connection to an Ethereum node
+var client *ethclient.Client
+var pk *ecdsa.PrivateKey
 
 func init() {
 	// TODO(prin-r): Initialize a connection to Ethereum
+	var err error
+	eth_rpc := os.Getenv("ETH_RPC")
+	client, err = ethclient.Dial(eth_rpc)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	pk, err = crypto.HexToECDSA(os.Getenv("PK"))
```

Comment:
If you are not planning to implement the no env case here. Please add `// TODO(prin-r): something`